.github/workflows/docs.yml

```diff
@@ -0,0 +1,26 @@
+name: Documentation
+
+on:
+  push:
+    branches: [ master ]
+
+  pull_request:
+    branches: [ master ]
```

Comment:
we shouldn't run this on PR contents, only on merges/pushes

Reply:
fixed